tools/tsp-client/src/commands.ts

```diff
@@ -31,6 +33,56 @@ import { sortOpenAPIDocument } from "@azure-tools/typespec-autorest";
 
 const defaultRelativeEmitterPackageJsonPath = joinPaths("eng", "emitter-package.json");
 
+// This function resolves for the correct emitter to use in a language repository based on whether
+// the user has provided an alternate dependency file path or if there are existing configuration
+// files in the repository. Supported configuration files are located at
+// <repo root>/eng/emitter-package.json or <repo root>/eng/tspclientconfig.yaml.
+// The function will return the emitter name and the path to the emitter package.json file relative
+// to repo root. If the default emitter-package.json is used, the path will be undefined.
```

Comment:
nit: should we use the standard multiline docs for function docs?

Reply:
Done